timemachine/cpp/src/math_utils.cuh

```diff
@@ -1,13 +1,3 @@
 #pragma once
 
-#include "surreal.cuh"
```

Comment:
Should we move `surreah.cuh` to `attic/` instead of deleting?

Reply:
I think that makes sense. It has some pedagogical utility (demonstrating how to implement forward mode AD via operator overloading).

Reply:
Resurrected and moved to `attic/` in 4847d60